package.json

```diff
@@ -63,7 +65,7 @@
     "route": "/acquisition-units",
     "okapiInterfaces": {
       "acquisitions-units": "1.1",
-      "users": "15.0"
+      "users": "15.1"
```

Comment:
there is a 15.0 in the release spreadsheet, maybe it makes sense to leave 15.0 as is https://docs.google.com/spreadsheets/d/11ciGyodrIX_NxjZmBYfDAp9mjfKcL_LSG32Qn-H0nkM/edit#gid=0

Reply:
in descriptor it's 15.1, seems nobody updated doc

Reply:
you can't say that it was released. They might have release 15.0 and updated version after) Since we still get the lates minor version of interface, it's safer to keep the previous